code/controllers/subsystem/air.dm

```diff
@@ -144,6 +220,24 @@ var/datum/subsystem/air/SSair
 			return
 
 
+/datum/subsystem/air/proc/process_excited_groups(resumed = 0)
+	if (!resumed)
+		src.currentrun = excited_groups.Copy()
+	//cache for sanic speed (lists are references anyways)
+	var/list/currentrun = src.currentrun
+	while(currentrun.len)
+		var/datum/excited_group/EG = currentrun[1]
+		currentrun.Cut(1, 2)
+		EG.breakdown_cooldown++
+		EG.dismantle_cooldown++
+		if(EG.breakdown_cooldown >= EXCITED_GROUP_BREAKDOWN_CYCLES)
```

Comment:
Using `>=` here makes it breakdown every tick after the threshold, not just the first time. Is this an intentional change in functionality?